runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -383,8 +386,7 @@ public void onChatMessage(ChatMessage event)
 		}
 
 		final String message = event.getMessage();
-
```

Comment:
Re-add this newline I guess

Reply:
OK you're right I didn't mean to delete that